src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/12.diff.sql

```diff
@@ -0,0 +1,271 @@
+﻿
+/****************************************************************************************
+Guidelines to create migration scripts - https://github.com/microsoft/healthcare-shared-components/tree/master/src/Microsoft.Health.SqlServer/SqlSchemaScriptsGuidelines.md
+
+This diff is broken up into several sections:
+ - The first transaction contains changes to tables and stored procedures.
+ - The second transaction contains updates to indexes.
+ - IMPORTANT: Avoid rebuiling indexes inside the transaction, it locks the table during the transaction.
+******************************************************************************************/
+SET XACT_ABORT ON
+
+BEGIN TRANSACTION
```

Comment:
Index creation should not be in a transaction, it will create lock for the whole process

Reply:
Done. Cleaned up